depmapomics/config_global.py

```diff
@@ -0,0 +1,741 @@
+import numpy as np
+
+
+LINES_TO_RELEASE = []
+SAMPLESETNAME = "22Q2test"
+RELEASE = SAMPLESETNAME.lower()
+
+CACHE_PATH = "~/.depmapomics/"
+TMP_PATH = "/tmp/"
+
+### GCP credentials
+SHEETCREDS = "../../.credentials.json"
```

Comment:
need to replace with ~ .  I know it bugs but you can get the location of ~ through other means